scripts/gen_eval_files.lua

```diff
@@ -211,6 +211,39 @@ local function get_api_meta()
   return ret
 end
 
+---@param text string
+local function fix_indent(text)
+  local lines = vim.split(text, "\n")
```

Comment:
The `stylua` config of this repo prefers single quotes.

Reply:
It seems that that file never was formatted with stylua. 
I don't want to format it now, since that would lead to an unreadable PR.
Once this one is merged, someone (or me) can make a new PR with just the re-formatted text